### ISSUE TITLE
Where is your sponsor button? ^_^

### ISSUE THREAD

**Issue by creatorrr** [hlissner/doom-emacs#1532]
Hey @hlissner , I am sure you get this enough but `doom-emacs` is *amazeballs*. Thank you for the amazing work and thoughtfulness you've clearly put into this project. Where is your sponsor button? (unless of course you don't like free money :P)

**Comment by creatorrr** [comment]
I am gonna close this just coz it probably takes away gh real estate from actual issues but kudos again! :)

**Comment by hlissner** [comment]
Thanks you! I'm happy to hear my insanity is useful to others. Github sponsors was mentioned on [our discord](https://discord.gg/bcZ6P3y) a couple times. I applied a week or so ago and am still wait listed, so we just have to wait and see!

> (unless of course you don't like free money :P)

It still blows my mind that people want to support Doom, but I've warmed up to the idea. If it means fewer hours freelancin' and more hours Emacsin', I can't complain!
